artiq/gateware/rtio/phy/ttl_simple.py

```diff
@@ -145,11 +145,18 @@ def __init__(self, pad):
 
 
 class ClockGen(Module):
-    def __init__(self, pad, ftw_width=24):
+    def __init__(self, pad, pad_n=None, ftw_width=24, dci=False):
         self.rtlink = rtlink.Interface(rtlink.OInterface(ftw_width))
 
         # # #
 
+        pad_o = Signal()
+        if pad_n is None:
+            self.comb += pad.eq(pad_o)
+        else:
+            self.specials += Instance("IOBUFDS",
```

Comment:
Use ``DifferentialOutput`` and remove redundant code in ``eem.py`` (``Urukul.add_std``).